client/auth/controller.js

```diff
@@ -20,16 +22,27 @@ import userFactory from 'lib/user';
 import Main from 'components/main';
 import PulsingDot from 'components/pulsing-dot';
 
-module.exports = {
+export default {
 
-	// Login screen used by the desktop application
-	login: function() {
-		if ( OAuthToken.getToken() ) {
-			page( '/' );
-		} else {
-			ReactDom.render(
-				React.createElement( LoginComponent, {} ),
-				document.getElementById( 'primary' )
+	login: function( context ) {
+		if ( config.isEnabled( 'oauth' ) ) {
+			// hack to keep the blue background for Desktop login
+			document.querySelector( '.layout' ).style.backgroundColor = '#0087be';
```

Comment:
@scruffian the login screen in the WordPress.com app is just more than a login, it's also a splash page since it's the first page the user sees when opening the app. Maybe @jasmussen can expand a bit on that matter.